rustls/src/crypto/ring/tls12.rs

```diff
@@ -278,6 +278,10 @@ impl MessageEncrypter for GcmMessageEncrypter {
 
         Ok(OpaqueMessage::new(msg.typ, msg.version, payload))
     }
+
+    fn encrypted_payload_len(&self, payload_len: usize) -> usize {
+        GCM_EXPLICIT_NONCE_LEN + payload_len + self.enc_key.algorithm().tag_len()
```

Comment:
very small nit: the other `encrypted_payload_len` impls are written with `payload_len` as the first term in the addition. It might be nice to match that here for consistency's sake.